[0.023s] Invoking command in '/home/sudheer/ros2_ws/build/my_robot_interfaces': CMAKE_PREFIX_PATH=/home/sudheer/ros2_ws/install/my_py_pkg:/opt/ros/humble /usr/bin/cmake --build /home/sudheer/ros2_ws/build/my_robot_interfaces -- -j4 -l4
[0.079s] -- Found ament_cmake: 1.3.8 (/opt/ros/humble/share/ament_cmake/cmake)
[0.268s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.271s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.275s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.316s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.589s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.589s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.589s]   my_robot_interfaces --arguments-file
[0.590s]   /home/sudheer/ros2_ws/build/my_robot_interfaces/rosidl_adapter__arguments__my_robot_interfaces.json
[0.590s]   --output-dir
[0.590s]   /home/sudheer/ros2_ws/build/my_robot_interfaces/rosidl_adapter/my_robot_interfaces
[0.590s]   --output-file
[0.591s]   /home/sudheer/ros2_ws/build/my_robot_interfaces/rosidl_adapter/my_robot_interfaces.idls)
[0.591s]   returned error code 1:
[0.592s] 
[0.592s]   Traceback (most recent call last):
[0.592s] 
[0.592s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.592s]       return _run_code(code, main_globals, None,
[0.592s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.592s]       exec(code, run_globals)
[0.592s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.592s]       sys.exit(main())
[0.592s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.592s]       abs_idl_file = convert_to_idl(
[0.592s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.592s]       return convert_srv_to_idl(
[0.593s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.593s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.593s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 859, in parse_service_string
[0.593s]       request_message = parse_message_string(
[0.593s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[0.593s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[0.593s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[0.593s]       self.base_type = BaseType(
[0.593s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.593s]       raise InvalidResourceName(
[0.593s] 
[0.593s]   rosidl_adapter.parser.InvalidResourceName: 'robot_Request' is an invalid
[0.593s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.593s] 
[0.594s] Call Stack (most recent call first):
[0.594s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.594s]   CMakeLists.txt:16 (rosidl_generate_interfaces)
[0.594s] 
[0.594s] [0m
[0.594s] -- Configuring incomplete, errors occurred!
[0.594s] See also "/home/sudheer/ros2_ws/build/my_robot_interfaces/CMakeFiles/CMakeOutput.log".
[0.601s] gmake: *** [Makefile:747: cmake_check_build_system] Error 1
[0.607s] Invoked command in '/home/sudheer/ros2_ws/build/my_robot_interfaces' returned '2': CMAKE_PREFIX_PATH=/home/sudheer/ros2_ws/install/my_py_pkg:/opt/ros/humble /usr/bin/cmake --build /home/sudheer/ros2_ws/build/my_robot_interfaces -- -j4 -l4
